docs/configuration/authentication/oidc_org.rst

```diff
@@ -0,0 +1,48 @@
+.. _configuration_authentication_oidc_org:
+
+=======================================
+OpenID Connect for organisation members
+=======================================
+
+Open Forms support login on forms by *organisation members* through Single Sign On (SSO) via the OpenID Connect protocol (OIDC).
```

Comment:
I checked the "organization" spelling:

```
organization: 16 hits / 7 files
organisation: 7 hits / 7 files
```

-> so let's stick to `organization` as that's used most often. I've updated the other files in cd5d9155b1a66197e256be9f614b48aec3567409